main.js

```diff
@@ -61,3 +61,23 @@ function scrollFunction() {
 function scrollToTop() {
   window.scrollTo(0, 0);
 }
+
+function showMenu(){
```

Comment:
you named the function "showMenu" but it can also close the menu as it uses the toggle method for all the classes used here - perhaps toggleMenu would be more accurate

Reply:
Thanks, have changed this now!